### ISSUE TITLE
Unable to continue training of a Multiple Layer bidirectional LSTM

### ISSUE THREAD

**Issue by 0n3m4v3r1ck** [googlecolab/colabtools#715]
Bug report for Colab: http://colab.research.google.com/.

For questions about colab usage, please use [stackoverflow](https://stackoverflow.com/questions/tagged/google-colaboratory/).

- Describe the current behavior:
Training on imdb_reviews/subwords8k stops unexpectedly and frequently in between. 
Epoch 1/10
W0824 04:34:29.570695 140670907217792 deprecation.py:323] From /usr/local/lib/python3.6/dist-packages/tensorflow/python/ops/math_grad.py:1250: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
391/391 [==============================] - 1829s 5s/step - loss: 0.6854 - accuracy: 0.5404 - val_loss: 0.0000e+00 - val_accuracy: 0.0000e+00
Epoch 2/10
391/391 [==============================] - 1546s 4s/step - loss: 0.5417 - accuracy: 0.7335 - val_loss: 0.5005 - val_accuracy: 0.7640
Epoch 3/10
391/391 [==============================] - 1463s 4s/step - loss: 0.4369 - accuracy: 0.8025 - val_loss: 0.5128 - val_accuracy: 0.7624
Epoch 4/10
321/391 [=======================>......] - ETA: 2:59 - loss: 0.3670 - accuracy: 0.8473

- Describe the expected behavior:
Training needs to continue and history variable should be populated so that history can be plotted too.

- The web browser you are using (Chrome, Firefox, Safari, etc.):
Chrome

- Link to self-contained notebook that reproduces this issue
  (click the *Share* button, then *Get Shareable Link*):
https://colab.research.google.com/drive/1HqmMCZxXkytxMM_DkedQQoCJCYuHZjvD
